ansys_optical_automation/application/example_VRLab_animation.py

```diff
@@ -0,0 +1,105 @@
+import csv
+import time
+import tkinter as tk
+from tkinter import filedialog
+
+from ansys_optical_automation.post_process.dpf_hdri_viewer import DpfHdriViewer
+
+
+def getfilename(extension, save=False):
+    """
+    Parameters
+    ----------
+    extension : str
+        containing the which file extension in *.ending format
+    save : Bool
+        option to define to open(default) or save. The default is False.
+
+    Returns
+    -------
+    str
+        string containing the selected file path
+    """
+    root = tk.Tk()
+    root.withdraw()
+    if not save:
+        file_path = filedialog.askopenfilename(filetypes=[("File", extension)])
+    else:
+        file_path = filedialog.asksaveasfilename(filetypes=[("File", extension)])
+        if not file_path.endswith(extension.strip("*")):
+            file_path += extension.strip("*")
+    return file_path
+
+
+def animation_run(speos_file, csv_file):
```

Comment:
i would suggest that this becomes a method of the HDRI viewer class as it might be quite intersting for everyone